### ISSUE TITLE
a11y: feat(bal-close): add title attribute

### ISSUE THREAD

**Issue by GinaBiondo** [baloise/design-system#946]
### Current Behavior

Icon Buttons and Links are currently without title.

### Expected Behavior

Buttons, links that are only Icons should give the possibility to set a title to meet a11y criteria. All components should be checked for missing title attributes.

### Steps to Reproduce

regular usage of bal-close in:

-bal-hint
-bal-snackbar
-bal-taost
-bal-modal

### Version

13.4.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
@GinaBiondo-aperto the alt attribute only exist on image elements not on button or labels. did you meant the title attribute?
